src/ctia/stores/es/sighting.clj

```diff
@@ -1,28 +1,74 @@
 (ns ctia.stores.es.sighting
-  (:require
-   [schema.core :as s]
-   [ctia.stores.es.crud :as crud]
-   [ctia.stores.es.query :refer [sightings-by-observables-query]]
-   [ctim.schemas.sighting :refer [Sighting
-                                  NewSighting
-                                  StoredSighting]]
-   [ctim.schemas.indicator :refer [Indicator]]
-   [ctia.lib.es.document :refer [search-docs]]))
-
-
-(def handle-create-sighting (crud/handle-create :sighting StoredSighting))
-(def handle-read-sighting (crud/handle-read :sighting StoredSighting))
-(def handle-update-sighting (crud/handle-update :sighting StoredSighting))
+  (:require [ctia.lib.pagination :refer [list-response-schema]]
+            [ctia.stores.es.crud :as crud]
+            [ctim.schemas
+             [common :refer [Observable]]
+             [sighting :refer [StoredSighting]]]
+            [schema-tools.core :as st]
+            [schema.core :as s]))
+
+(s/defschema ESStoredSighting
+  (st/merge StoredSighting
+            {:observables_hash [s/Str]}))
+
+(def es-coerce! (crud/coerce-to-fn [(s/maybe ESStoredSighting)]))
+
+(s/defn observable->observable-hash :- s/Str
+  [{:keys [type value] :as obsrvable :- Observable}]
```

Comment:
- the `:as obsrvable` is not necessary

- `type` conflict with `clojure.core/type` why not `observable-type`?